modules/ztp-troubleshooting-ztp-gitops-supermicro-tls.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-troubleshooting-ztp-gitops-supermicro-tls_{context}"]
+= Troubleshooting {ztp} virtual media booting on Supermicro servers
+
+SuperMicro X11 servers do not support virtual media installations when the image is served using the `https` protocol. As a result, {sno} deployments for this environment fail to boot on the target node. To avoid this issue, log in to the hub cluster and disable TLS in the `Provisioning` resource. This ensures the image is not served with TLS even though the image address uses the `https` scheme. 
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Disable TLS in the `Provisioning` resource by running the following command, 
++
+[source,terminal]
+----
+$ oc patch provisioning provisioning-configuration --type merge -p '{"spec":{"disableVirtualMediaTLS": true}}'
+----
+
+. Continue the steps to deploy your {sno} cluster.
```

Comment:
Only suggestion would be to possibly direct reader to those steps via link..but just a suggestion :)